libraries/aws_db_parameter_groups.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'aws_backend'
+
+class AwsDbParameterGroups < AwsResourceBase
+  name 'aws_db_parameter_groups'
+  desc 'Verifies settings for a Parameter Group in bulk'
+  example '
+    describe aws_db_parameter_groups do
+      it { should exist }
+    end
+  '
+
+  attr_reader :table, :api_response
+
+  FilterTable.create
+             .register_column(:db_parameter_group_names,                field: :db_parameter_group_name)
+             .register_column(:db_parameter_group_families,             field: :db_parameter_group_family)
+             .register_column(:descriptions,                            field: :description)
+             .register_column(:db_parameter_group_arns,                 field: :db_parameter_group_arn)
+             .install_filter_methods_on_resource(self, :table)
+
+  def initialize(opts = {})
+    super(opts)
+    validate_parameters
+    @table = fetch_data
+  end
+
+  def fetch_data
+    parameter_group_rows = []
+    pagination_options = {}
+    loop do
+      catch_aws_errors do
+        @api_response = @aws.rds_client.describe_db_parameter_groups(pagination_options)
+      end
+      return [] if !api_response || api_response.empty?
+
+      api_response.db_parameter_groups.each do |parameter_group|
```

Comment:
since there is no manipulations of the values here. You could simplify this as below, 
```suggestion
     parameter_group_rows += api_response.db_parameter_groups.map(&:to_h)
```

Reply:
Done and tested